packages/labs/ssr/src/lib/dom-shim.ts

```diff
@@ -23,10 +23,12 @@ import fetch from 'node-fetch';
  *  generally only be true when adding window to a fresh VM context that
  *  starts with nothing.
  * @param props Additional properties to add to the window global
+ * @param exposeWindow Whether the `window` object should be exposed globally
```

Comment:
The team had a discussion and we are okay making no `window` the only behavior for the DOM shim. It's easy enough for users to add `globalThis.window = globalThis` on their own if they need to so we don't really even need an option.